src/patternfly/components/Alert/docs/code.md

```diff
@@ -6,14 +6,24 @@ Always add a modifier class. Do not use `.pf-c-alert` on its own.
 
 | Attribute | Applied To | Outcome |
 | -- | -- | -- |
-| `role="alert"` | `.pf-c-alert` |  Brings immediate attention to the user. |
+| `role="alert"` | `.pf-c-alert` |  Brings immediate attention to the user. This should only be used on dynamically created alerts such as at the top of form when there are errors in the form. |
+| `aria-label="Success Notification"` | `.pf-c-alert` |  Indicates the Success Notification. |
+| `aria-label="Danger Notification"` | `.pf-c-alert` |  Indicates the Danger Notification. |
+| `aria-label="Warning Notification"` | `.pf-c-alert` |  Indicates the Warning Notification. |
+| `aria-label="Information Notification"` | `.pf-c-alert` |  Indicates the Information Notification. |
+| `aria-hidden="true"` | `.pf-c-alert__icon` |  Hides icon for assitive technologies. |
+
+| Class | Applied To | Outcome |
+| -- | -- | -- |
+| `.sr-only` | `.pf-c-alert__title <span>` | Content that is visually hidden but accessible to assitive technologies. |
```

Comment:
There's a typo with "assitive".

Also, in this section, should we state that this element should include the type of alert/notification (i.e. Success, Warning...)?